diff -up lmms/CMakeLists.txt~ lmms/CMakeLists.txt
--- lmms/CMakeLists.txt~	2018-11-03 02:43:42.000000000 +0100
+++ lmms/CMakeLists.txt	2021-05-23 15:05:43.959322349 +0200
@@ -564,6 +564,9 @@ IF(USE_CCACHE)
     ENDIF()
 ENDIF()
 
+# pass LIB_SUFFIX
+add_definitions(-DLIB_SUFFIX="${LIB_SUFFIX}")
+
 # make sub-directories
 ADD_SUBDIRECTORY(cmake)
 ADD_SUBDIRECTORY(src)
diff -up lmms/src/core/PluginFactory.cpp~ lmms/src/core/PluginFactory.cpp
--- lmms/src/core/PluginFactory.cpp~	2018-11-03 02:43:42.000000000 +0100
+++ lmms/src/core/PluginFactory.cpp	2021-05-23 15:24:24.965029963 +0200
@@ -64,7 +64,7 @@ PluginFactory::PluginFactory()
 	//           plugins at "C:/Program Files/LMMS/plugins/"
 
 #ifndef LMMS_BUILD_WIN32
-	addRelativeIfExists("../lib/lmms"); // Installed
+	addRelativeIfExists("../lib" LIB_SUFFIX "/lmms"); // Installed
 #endif
 	addRelativeIfExists("plugins"); // Portable
 #ifdef PLUGIN_DIR // We may also have received a relative directory via a define
